### DISCUSSION TITLE
Issue with AKS Velero Setup

### DISCUSSION THREAD

**Discussion by skri547** [vmware-tanzu/velero#4533] [Community support Q&A]
Hi Team, I have used the below command to install the velero on AKS Cluster

1. I am using the setup with Service principal method which has access to the Resource Group, and installed velero with the below command by following this doc - https://github.com/vmware-tanzu/velero-plugin-for-microsoft-azure 

velero install \
    --provider azure \
    --plugins velero/velero-plugin-for-microsoft-azure:v1.3.0 \
    --bucket $STORAGE_CONTAINER_NAME \
    --secret-file ./credentials-velero \
    --backup-location-config resourceGroup=$AZURE_RESOURCE_GROUP,storageAccount=$STORAGE_ACCOUNT,subscriptionId=$AZURE_SUBSCRIPTION_ID \
    --snapshot-location-config resourceGroup=$AZURE_RESOURCE_GROUP,subscriptionId=$AZURE_SUBSCRIPTION_ID
	

And All the resources got deployment and I see some errors in the pods logs when checked, attached below 
<img width="958" alt="image" src="https://user-images.githubusercontent.com/68077126/149607089-634188a3-4467-4f5c-8373-b2224362c793.png">

I am trying to exec in to pod but its not letting me in. 
<img width="792" alt="image" src="https://user-images.githubusercontent.com/68077126/149606704-fd885e3c-6f97-4381-86e7-d27cc13bfa8b.png">

If you could please provide me the solution to the problem that would be great. Let me know if you want to share any other details

**Comment by chefcook** [comment]
I'm pretty sure you can't get a shell to run because this is a distroless container image.
